src/restore.c

```diff
@@ -333,30 +393,30 @@ restore_main(int32_t argc, char **argv)
 		}
 	}
 
-	if (!conf.validate && !batch_uploader_await(&status.batch_uploader)) {
+	if (!conf->validate && !batch_uploader_await(&status->batch_uploader)) {
 		res = EXIT_FAILURE;
 	}
 
 	// NOTE this is here to support the --indexes-last option
```

Comment:
Tangent. I wonder why --indexes-last is even needed. I would think indexes-last should always be true

Reply:
Why? Do you get an error if nothing exists for them to index?

Reply:
When you write a record and an index already exists it takes some time for the record to be indexed by the sindex. If you write the record first then create the sindex all existing records are immediately indexable

Reply:
Hmmm, that does seem like it should be the default. I'm going to make a ticket for it.

Reply:
Made TOOLS-2675 to track this